.github/workflows/release.yml

```diff
@@ -1,26 +1,116 @@
 name: Release
 
 on:
-  push:
-    tags:
-      - '[0-9]+.[0-9]+.[0-9]+'
+  workflow_dispatch:
+    inputs:
+      releaseType:
+        type: choice
+        options:
+          - minor
+          - patch
+        default: minor
+        required: true
+        description: 'minor: v0.X.0, patch: v0.0.X'
+      debug:
+        type: boolean
+        default: true
+        description: 'executes the workflow in debug mode (skip publishing tag, release and docker image)'
```

Comment:
Nitpick: Now that the order of the steps was changed, can we also swap in this sententce? 
```
        description: 'executes the workflow in debug mode (skip the publishing tag, docker image and release steps)'
```

Reply:
Thanks. https://github.com/prebid/prebid-cache/pull/135/commits/cfb0f0c8baac83e7551afb5bd5cd6849716218dc updates the description